samples/bundle-war-deploy/demo-war/pom.xml

```diff
@@ -41,6 +31,7 @@
     <dependency>
       <groupId>org.apache.geronimo.specs</groupId>
```

Comment:
To be honest I would tend to use `javax.servlet:javax.servlet-api` (at the right version for Liberty) to avoid using an implementation-specific artifact.